posthog/tasks/status_report.py

```diff
@@ -151,14 +151,21 @@ def status_report(*, dry_run: bool = False) -> Dict[str, Any]:
 def capture_event(name: str, report: Dict[str, Any], dry_run: bool) -> None:
     if not dry_run:
         posthoganalytics.api_key = "sTMFPsFhdP1Ssg"
-        posthoganalytics.capture(get_machine_id(), name, {**report, "scope": "machine"})
+        posthoganalytics.capture(get_machine_id(), name, {**report, "scope": "machine"}, groups={"instance": SITE_URL})
+
+        if "instance_usage_summary" in report:
+            posthoganalytics.group_identify("instance", SITE_URL, fetch_instance_params(report))
 
         for user in User.objects.all():
             posthoganalytics.capture(user.distinct_id, f"user {name}", {**report, "scope": "user"})
     else:
         print(name, json.dumps(report))  # noqa: T001
 
 
+def fetch_instance_params(report: Dict[str, Any]) -> dict:
+    return {"site_url": SITE_URL, "machine_id": get_machine_id(), **report["instance_usage_summary"]}
```

Comment:
I would keep the `report["instance_usage_summary"]` params you have but without the params that end in `_in_period`, but non-blocking. I would also get rid of `machine_id` as I don't think this will be robust enough to be useful.